gnomad/utils/filtering.py

```diff
@@ -436,6 +441,10 @@ def filter_to_gencode_cds(
     :param gencode_ht: Gencode Table to use for filtering the input Table/MatrixTable
         to CDS regions. Default is None, which will use the default version of the
         Gencode Table resource.
+    :param genes: Optional gene(s) to filter to. Can be a single gene string or list of genes. Default is None.
+    :param by_gene_symbol: Whether to filter by gene symbol. Default is True. If False, will filter by gene ID.
+    :param padding_bp: Number of bases to pad the CDS intervals by. Default is 0.
+    :param max_intervals: Maximum number of intervals to collect before filtering. Default is 1000.
```

Comment:
```suggestion
    :param max_collect_intervals: Maximum number of intervals for the use of `hl.filter_intervals` for filtering. When the number of intervals to filter is greater than this number, `filter`/`filter_rows` will be used instead. The reason for this is because `hl.filter_intervals` is faster, but when the number of intervals is too large, this can cause memory errors. Default is 3000.
```